src/app/pad.component.scss

```diff
@@ -77,6 +77,55 @@
           margin-left: 16px;
         }
 
+        .keymap-button {
```

Comment:
instead of copy/pasting the mode button directly can you make parts that are actually identical into a mixin (or just one common style that both sets of elements can match directly if they actually are identical?)